docs/README.rst

```diff
@@ -0,0 +1,145 @@
+.. _readme:
+
+resharper-formula
+=================
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/resharper-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/resharper-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+Formula to install Resharper on Windows
```

Comment:
Only `on Windows`?  I'll merge for now anyway but this needs to be updated.

Reply:
Yes, its only Windows software last time I checked.